sdcm/cluster_k8s/__init__.py

```diff
@@ -1036,17 +1036,20 @@ def deploy_ingress_controller(self, pool_name: str = None):
                 memory_limit = convert_memory_units_to_k8s_value(memory_limit)
                 cpu_request = convert_cpu_units_to_k8s_value(cpu_request)
                 memory_request = convert_memory_units_to_k8s_value(memory_request)
+                haproxy_maxconn = 70000
             else:
-                cpu_limit = '100m'
-                memory_limit = '50M'
+                cpu_limit = '400m'
+                memory_limit = '200M'
```

Comment:
Where these numbers come from? Is it calculated?

Reply:
mostly trial and error, before we didn't had limits, only request
I've just it's just request x 4.

Reply:
Please, add a comment above something like following:
```
# TODO: measure these values in performance testing and update if needed.
```